Avatar_/avatar/avatar.css

```diff
@@ -0,0 +1,29 @@
+@import url('/css/style.css');
+.avatar.larger{
+    border-radius: 50%;
+    border: 0.05rem solid black;
+    margin: 1rem;
+    width: 6rem;
+    height: 6rem;
+}
+.avatar.large{
+    border-radius: 50%;
+    border: 0.05rem solid black;
+    width: 5rem;
+    height: 5rem;
+    margin: 1rem;
+}
+.avatar.medium{
+    border-radius: 50%;
+    border: 0.05rem solid black;
+    width: 4rem;
+    height: 4rem;
+    margin: 1rem;
+}
```

Comment:
You can give border & radius, margin, border to all Avatar together as u repeating same.

Reply:
Yes done according to your suggestions